### ISSUE TITLE
Create a GitHub app to list all pull requests in my repositories

### ISSUE THREAD

**Issue by egoist** [egoist/ideas#9]
Because this is a useful yet missing feature on GitHub (correct me if I'm wrong).

Probably use https://probot.github.io to build this.

**Comment by MarchWorks** [comment]
With electron? or a website

**Comment by tanvirraj** [comment]
can you explain more, i mean list all PR in all repo or single repo

**Comment by kayandra** [comment]
I don't know how I feel about this idea when Github's already building an app

**Comment by egoist** [comment]
@kayandra It doesn't have to be a mobile app, could be a simple web app. And GitHub's mobile app doesn't seem to have this feature either.

**Comment by kayandra** [comment]
Yeah, I just signed up for the beta and it doesn't have it. Wishing you all the success in the world 🤗
